templates/post_type.mustache

```diff
@@ -1,5 +1,5 @@
 	register_post_type( '{{slug}}', array(
-		'labels'            => array(
+		'labels'                => array(
```

Comment:
Either these white spaces changes need to be reverted or the tests in `features/scaffold.feature:140/174/182/190` adjusted...

Reply:
@gitlost  working on it thanks for the help